packages/js/internal-style-build/index.js

```diff
@@ -8,6 +8,26 @@ const RemoveEmptyScriptsPlugin = require( 'webpack-remove-empty-scripts' );
 const postcssPlugins = require( '@wordpress/postcss-plugins-preset' );
 const StyleAssetPlugin = require( './style-asset-plugin' );
 
+/**
+ * @deprecated intended for React version migration and can be dropped from exports any time without further notice.
+ */
+const ForkTsCheckerWebpackPlugin = require( 'fork-ts-checker-webpack-plugin' );
+
+/**
+ * https://github.com/woocommerce/woocommerce/pull/47486: do not break build if TS emits errors - React 18 migration related.
+ * The solution is based https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/issues/232#issuecomment-1322651312.
```

Comment:
nitpick  "is based on"

Reply:
Thanks, updated in a8bcdffeebc1f02c4145bfd73317c0d11912a3c8.